### ISSUE TITLE
Code generation fails if schema file directory is symbolic linked

### ISSUE THREAD

**Issue by konomae** [uhyo/nitrogql#14]
Hi, I'm interested in trying this tool.
But I found a problem about symbolic link. 👀
(This situation is a simplified version of monorepo with pnpm.)

### Directory structure

```
.
├── api
│   └── schema.graphql
├── graphql.config.yaml
├── node_modules
│   └── api -> ../api # symbolic link
└── query.graphql
```

### graphql.config.yaml

```yaml
schema: node_modules/api/schema.graphql # doesn't work
# schema: api/schema.graphql # works
documents: query.graphql
extensions:
  nitrogql:
    generate:
      schemaOutput: "./generated/schema.ts"
```

### query.graphql

```graphql
query Ping {
  ping
}
```

### schema.graphql

```graphql
type Query {
  ping: String!
}
```

### command

```
npx @nitrogql/cli@1.0.0-beta.6 generate

Found 1 error in operations:
/Users/user/nitro-repro-simple/query.graphql:1:1
query Ping {
^
Type 'Query' is not defined
  ping
}


Error in command 'generate':
```

**Comment by uhyo** [comment]
Thanks for the report! I can reproduce the issue. Investigating now :eyes:

**Comment by uhyo** [comment]
📝 Memo: Node's `path_filestat_get` returns 76 (notcapable) for `node_modules/api`

**Comment by uhyo** [comment]
https://github.com/nodejs/node/issues/49107

Probable I need to implement an alternative WASI implementation for this. 🥲

**Comment by konomae** [comment]
Thanks for investigating!
I'll watch for the node's issue. 👀

**Comment by uhyo** [comment]
@konomae Thank you for your patience! This should be fixed in `v1.0.0-beta.7`.

**Comment by konomae** [comment]
Thanks for fixing this!
